appsScript/CreateFormFields.js

```diff
@@ -0,0 +1,304 @@
+/**
```

Comment:
Looks good. you have a bit of repeat code, i'm wondering if you could externalise some, with something like:
```

setTextItem(title, helpText, required){
    textItem = form.addTextItem()
    textItem.setTitle(title)
    textItem.setHelpText(helpText)
    textItem.setRequired(required)
   return textItem
}
```

And the same for listItems and multipleChoiceItems. Then you just call the helper function for each of the form entries.